ADA/ada.qmd

```diff
@@ -36,14 +35,65 @@ The ambition is to enable access to all datasets across DfE and ESFA and retire
 
 ---
 
+# What is Databricks?
+
+Databricks is a browser-based platform developed by Microsoft which allows you to use notebooks to access and analyse data. You can also use it for machine learning. Databricks notebooks support multiple programming languages, including R, Python, and SQL. Unlike RStudio or SQL Server Management Studio (SSMS), you don't have to download any software to be able to use it. You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can check the status of various datasets [in the data catalogue dashboard](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) 
+
+Accessing data through Databricks works slightly differently to the way that you might currently access your data in SSMS. You can read more about how Databricks works, including data storage and how to use clusters, on the [ADA guidance site](https://analytical-data-access.azurewebsites.net/get-support/how-databricks-works). 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks)
+
+One benefit of Databricks is the use of workflows to run your SQL code. Workflows allow you to automate the running of single or multiple SQL scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+---
+
+# How will I know when I need to move to Databricks?
```

Comment:
I think we're missing something in the terms of strategy for us / what we're pushing, we could put an info box with a recommendation or something of what we suggest teams do. For example, we could recommend teams engage earlier so that they have more time to dual run, or we could suggest they hold off until approach to save wasting time while ADA fix up more issues?

Reply:
I'm not sure this steer should come from us, but happy to discuss it!

Reply:
I think I've addressed this in one of my earlier comments about when data bricks will impact me. I've suggested it comes much higher up on the page as part of the intro and is kept brief - with space for ADA team to add more specifics.

You could end that section with something like, as always we encourage analysts to engage early, do the training and ask questions to ensure they are prepared ahead of having to use it.

Reply:
I think at a minimum we should put some kind of recommendation / steer for stats production teams (I'd apply that to all of the page, not just this section) as ultimately we do drive the strategy there and our unit is here to steer so we get consistency of approach. 

For analysis more widely we should probably be more cautious, Laura's suggestion works for that part.